var/spack/repos/builtin/packages/llvm-flang/package.py

```diff
@@ -16,7 +16,8 @@ class LlvmFlang(CMakePackage, CudaPackage):
 
     maintainer = ['naromero77']
 
-    version('master', branch='master')
+    version('release_90', branch='release_90')
```

Comment:
I'm pretty sure 20190329 will still be installed by default. Is this what you want?

Reply:
Thanks for pointing this out. I think its fine for now, though admittedly its a bit confusing. No one would install `llvm-flang` for its own sake. This package will probably go away in the near future because the developers are creating a monorepo for the classic flang. So in summary, I think its OK as is.